source/docs/04-cmn.xml

```diff
@@ -256,13 +256,13 @@
                            <item>octave (using @oct)</item>
                            <item>duration (using @dur)</item>
                         </list>
-                        <p>A typical note, in this case a quarter note C4, is therefore encoded like so: <figure>
+                        <p>A single note, in this case a quarter note C4, is therefore encoded like so: <figure>
                               <head/>
                               <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/cmn/cmn-sample098.txt" parse="text"/></egXML>
                            </figure>
                         </p>
                         <p>Because these attributes may not be required in all situations (such as <att>dur</att> for the notes of a chord), processing software should anticipate retrieving the information that would have been provided by missing attributes from a preceding note or <gi scheme="MEI">chord</gi> parent in the same <gi scheme="MEI">layer</gi>. Only information from <att>pname</att>, <att>oct</att> and <att>dur</att> attributes can be gathered in this fashion. No other attributes can be treated this way.</p>
-                        <p>The default values for <att>pname</att> and <att>oct</att> conform to the <ref target="https://en.wikipedia.org/wiki/Scientific_pitch_notation">Acoustical Society of America representation for pitch name</ref>; that is, the letters A - G, albeit in lower case, and the numbers 0 - 9. The value for <att>oct</att> changes between B and C.</p>
+                        <p>The default values for <att>pname</att> and <att>oct</att> conform to the <ref target="https://en.wikipedia.org/wiki/Scientific_pitch_notation">Scientific Pitch Notation (SPN)</ref> (also known as American Standard Pitch Notation); that is, the letters A–G indicate the musical note name of a pitch, and the numbers 0–9 indicate the octave range to which a note belongs. <att>pname</att> values differ from this convention only by using lower case values for pitches (a–g instead of A-G). The value for <att>oct</att> changes between B and C, that is, octave ranges go from C, D, …, G, A, to B. For example, the so-called <hi rend="italic">middle C</hi> or <hi rend="italic">c'</hi> (the C in the middle, i.e. fourth C key from left, on a standard 88-key piano keyboard) is represented on the first ledger line in G clef notation and labelled as <hi rend="italic">C4</hi> in the naming convention of SPN, the note one semitone below would be labelled <hi rend="italic">B3</hi>, and <hi rend="italic">A4</hi> would refer to the first A above <hi rend="italic">C4</hi>.</p>
```

Comment:
```suggestion
                        <p>The default values for <att>pname</att> and <att>oct</att> conform to the <ref target="https://en.wikipedia.org/wiki/Scientific_pitch_notation">Scientific Pitch Notation (SPN)</ref> (also known as American Standard Pitch Notation); that is, the letters A–G indicate the musical note name of a pitch, and the numbers 0–9 indicate the octave range to which a note belongs. <att>pname</att> values differ from this convention only by using lower case values for pitches (a–g instead of A-G). The value for <att>oct</att> changes between B and C, that is, octave ranges go from C, D, …, G, A, to B. For example, <hi rend="italic">middle C</hi> or <hi rend="italic">c'</hi> (the C in the middle, i.e. fourth C key from left, on a standard 88-key piano keyboard) is represented on the first ledger line in G clef notation and labelled as <hi rend="italic">C4</hi>. In the naming convention of SPN, the note one semitone below would be labelled <hi rend="italic">B3</hi>, and <hi rend="italic">A4</hi> would refer to the first A above <hi rend="italic">C4</hi>.</p>
```

Reply:
The diff doesn't really help with the suggestion. What I suggested is removing "so-called" (it's common enough practice to call it middle C that we don't need to caveat it), and splitting the "For example" sentence as it was starting to run on.  So:

"...first ledger line in G clef notation and labelled as C4 in the naming convention..."

becomes

"...first ledger line in G clef notation and labelled as C4. In the naming convention..."

Reply:
I checked once more and there seems to be everything OK with the formatting, but a kind of length maximum for Github GUI's diff highlighting mechanism.

Reply:
Many thanks!